internal/featuresupport/feature_support_test.go

```diff
@@ -196,18 +196,42 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 			}
 			for _, platform := range platforms {
 				p := platform
-				list := GetFeatureSupportList("dummy", nil, &p)
+				list := GetFeatureSupportList(SupportLevelFilters{
+					OpenshiftVersion:     "dummy",
+					CPUArchitecture:      nil,
+					PlatformType:         &p,
+					HighAvailabilityMode: nil,
+				})
 				Expect(len(list)).To(Equal(16))
 			}
 		})
+		It("GetFeatureSupportList 4.12 with HighAvailabilityMode", func() {
+			list := GetFeatureSupportList(SupportLevelFilters{
+				OpenshiftVersion:     "4.12",
+				CPUArchitecture:      nil,
+				PlatformType:         nil,
+				HighAvailabilityMode: swag.String(models.ClusterHighAvailabilityModeNone),
```

Comment:
do we have another test with ClusterHighAvailabilityModeFull that should return a different length number of the list?

Reply:
If `HighAvailabilityMode` is set, it doesn't matter what is the value